docs/usage.md

```diff
@@ -29,6 +29,18 @@ system:
 sudo curl -L --output /usr/local/bin/pie https://github.com/php/pie/releases/latest/download/pie.phar && sudo chmod +x /usr/local/bin/pie
 ```
 
+### Docker installation
+
+PIE is published as binary-only Docker image, so you can install it easily during your Docker build:
+
+```Dockerfile
+COPY --from=ghcr.io/php/pie:latest /pie /usr/bin/pie
+```
+
+Instead of `latest` you can also use explicit versions like `x.y.z`, `x.y` or `x`, depending on stability level you want to achieve.
+
+If you want to verify signature of the PHAR, you need to additionally download `*.asc` file from [corresponding release](https://github.com/php/pie/releases).
```

Comment:
We have done away with the GPG signing now; this PR will need updating to use this for generating build provenance of this container image: https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds#generating-build-provenance-for-container-images

Reply:
Attestation step added, [here's the result](https://github.com/Wirone/php-pie/attestations/3509753).